rinja/src/html.rs

```diff
@@ -1,71 +1,203 @@
-use std::fmt;
-use std::num::NonZeroU8;
+use std::{fmt, str};
 
+#[cfg(not(any(target_arch = "x86", target_arch = "x86_64")))]
 #[allow(unused)]
 pub(crate) fn write_escaped_str(mut fmt: impl fmt::Write, string: &str) -> fmt::Result {
-    let mut escaped_buf = *b"&#__;";
+    // Even though [`jetscii`] ships a generic implementation for unsupported platforms,
+    // it is not well optimized for this case. This implementation should work well enough in
+    // the meantime, until portable SIMD gets stabilized.
+
+    // Instead of testing the platform, we could test the CPU features. But given that the needed
+    // instruction set SSE 4.2 was instroduced in 2008, that it has an 99.61 % avalibility rate
```

Comment:
```suggestion
    // instruction set SSE 4.2 was introduced in 2008, that it has an 99.61 % availability rate
```

This word (avalibili thing) is super hardcore to write... Took me a few tries to get it right. XD